apps/website/docs/infrastructure/azure/apim/api-access-policies.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_label: Manage API Access and Policies
+sidebar_position: 1
+---
+
+# Manage API Access and Policies
+
+This documentation outlines a few best practices for software engineers using
+APIM.
+
+The goal is to:
+
+- Improve operational simplicity
+- Increase domain separation and autonomy
+- Improve security
```

Comment:
issue: it looks like this document contains content unrelated to "Manage API Access and Policies" like versions

Reply:
I'd rather rename the page to a generic `Managing APIs and Policies`

Reply:
The entire paragraph is deceptive. First, it states that there are three different ways to establish a version (related to API management capabilities). Then, it says we recommend a specific way (related to REST API guidelines) without any clue. We should wait for API guidelines before publishing how to use the versions.

Reply:
I could write some doc about API guidelines, but I believe @lucacavallaro or @kin0992 could help more than me

Reply:
I agree with decoupling the APIM-related stuff from what should be related to the REST API.
In this document, I think we should remove the suggestions that are highly REST API related (like the versioning), and keep other suggestions that are more useful to our work environment (like the API test in production).

We could also talk about the priority of some REST API Guidelines

Reply:
@Krusty93 I agree with the importance of releasing such documentation, and I would remove the paragraph about versioning for the considerations made in the previous comments

Reply:
Removed versions and added an example of rate limit policy: https://github.com/pagopa/dx/pull/514/commits/8d59c3ae6b5af70c9d29137b0c56d1cdee4ffd44